### PR TITLE
separated getFilters and getViolation and added tests.

### PR DESCRIPTION
**Pull request by Wmaileq**
## Description of the change

- downloads prettified
- getFilters and getViolation separated

## Type of change

- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Part of #25 

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

**Comment by jessex** [comment]
Hey, @jovergaag: would you mind taking a pass through this code review when you get a chance? It's another boost in test coverage for the app.

File: src/assets/scripts/utils/getViolation.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { humanReadableTitleCase } from "../../../utils/transforms/labels";
+
+function getViolation({ reportedViolations, violationType }) {
+  console.log(reportedViolations, violationType);
```
Comment (jovergaag) [inline src/assets/scripts/utils/getViolation.js:4]: oops!

File: src/assets/scripts/utils/getFilters.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+const formatSupervisionType = (supervisionType) =>
+  supervisionType === "All"
+    ? "All supervision types"
+    : `Supervision type: ${humanReadableTitleCase(supervisionType)}`;
+
+const formatSupervisionLevel = (supervisionLevel) =>
+  supervisionLevel === "All"
+    ? "All supervision levels"
+    : `Supervision type: ${pipe(
```
Comment (jovergaag) [inline src/assets/scripts/utils/getFilters.js:40]: Should this be "Supervision level: "?

File: src/assets/scripts/utils/downloads.js [hunk 3]
```diff
@@ -15,21 +15,17 @@
...
-import pipe from "lodash/fp/pipe";
-import find from "lodash/fp/find";
-import get from "lodash/fp/get";
-
-import { SUPERVISION_LEVELS } from "../../../components/charts/new_revocations/ToggleBar/options";
-import { timeStamp } from './time';
-import infoAboutChart from '../../../utils/charts/info';
-import JSZip from 'jszip';
-import { humanReadableTitleCase } from '../../../utils/transforms/labels';
+import downloadjs from "downloadjs";
```
Comment (jovergaag) [inline src/assets/scripts/utils/downloads.js:18]: Thanks for cleaning this up!

File: src/assets/scripts/utils/__tests__/getFilters.js [hunk 4]
```diff
@@ -38,12 +39,18 @@ describe("#getFilters", () => {
       district: "SLCRC",
       chargeCategory: "SEX_OFFENCE",
       supervisionType: "PROBATION",
+      supervisionLevel: "ENHANCED",
     };
 
     const expected =
-      "3 years, District: SLCRC, Case type: Sex Offence, Supervision type: Probation";
+      "3 years, District: SLCRC, Case type: Sex Offence, Supervision type: Probation, Supervision type: Enhanced Supervision";
```
Comment (jovergaag) [inline src/assets/scripts/utils/__tests__/getFilters.js:46]: I believe this addition should be "Supervision level"

**Review by jovergaag — CHANGES_REQUESTED** [review]
Just one small copy change but this looks great! Thanks!

File: .eslintignore [hunk 5]
```diff
@@ -14,6 +14,8 @@ public/
 !src/App.js
 !src/index.js
 !src/assets/scripts/utils/__tests__/**/*.js
+!src/assets/scripts/utils/getFilters.js
+!src/assets/scripts/utils/getViolation.js
```
Comment (jovergaag) [inline .eslintignore:18]: Should we whitelist `downloads` now that it is prettified?

Comment (Wmaileq) [reply .eslintignore:18]: Not yet, there are a lot of eslint errors here that should be fixed while separating\writing tests on downloads functions.

File: src/assets/scripts/utils/downloads.js (hunk 3 above)
Comment (Wmaileq) [reply src/assets/scripts/utils/downloads.js:18]: It was just prettified, the next step should be fixing all eslint errors (15 problems)

**Review by jovergaag — APPROVED** [review]
💯

**Comment by Wmaileq** [comment]
@jovergaag, @jessex Please merge this one, as i have no rights.

**Pull request merged**